cinch/group_vars/all

```diff
@@ -8,3 +8,20 @@ repository_defaults:
   enabled: true
   skip_if_unavailable: false
   gpgcheck: false
+
+# Override these variables with an array of files to be uploaded as-is to the
+# destination host. Files in pre_upload_files will be uploaded before any other
+# code is run. Files in post_upload_files will be uploaded after all other tasks
+# have been run. Each element in the array must contain an object with the
+# following fields:
+# src (required): the local path to the file to upload
```

Comment:
Adding a commented syntax example of pre/post_upload_files usage with perhaps one file and one directory might be helpful to assist users in getting started here.